documentation/asciidoc/computers/config_txt/video.adoc

```diff
@@ -826,13 +826,13 @@ These values are valid if `hdmi_group=1` (CEA):
 | 4096x2160
 | 50Hz
 | 256:135
-| Pi 4
+| Pi 4<<super1>>
```

Comment:
`super1` is a bit of a meaningless link-name. `needsoverclock` or similar might be better?

Reply:
Fixed